app/routes/_gcn.circulars.edit.$circularId/CircularEditForm.tsx

```diff
@@ -0,0 +1,353 @@
+/*!
```

Comment:
Breaking out CircularEditForm into its own module is a good refactoring, period. Given that this PR has turned out to be quite complicated, this refactoring would be a good save point to get onto the main branch ASAP. That way, if any other changes occur to this form before the present PR is merged, there is less for you to rework. Would you please open a separate PR to do that refactoring?

Reply:
I think I caught all the references in this one: